pkg/plugins/scms/github/scm.go

```diff
@@ -140,7 +140,16 @@ func (g *Github) CreateCommit(workingDir string, commitMessage string) error {
 		return err
 	}
 
-	repositoryName := fmt.Sprintf("%s/%s", g.Spec.Owner, g.Spec.Repository)
+	if g.nativeGitHandler.IsForceReset() {
```

Comment:
I am wondering if it's an operation we should only do if committing via the API is configured.
Otherwise, we stick to the default behavior similar to the other scm

Reply:
We can just add a second condition.
What do you think?